tests.sh

```diff
@@ -128,5 +128,34 @@ test_template_local_path() {
   assertErrorCode 0 -p examples/moon-base template ../custom-template.tmpl -o Dockerfile-custom
 }
 
+test_run_repo_say_branch() {
+  result=$(./shuttle -p examples/repo-project-branched run say)
+  if [[ ! "$result" =~ "something clever" ]]; then
+    fail "Expected output to contain 'something minor', but it was:\n$result"
```

Comment:
The `if` looks for `something clever` but this error message states `something minor`.

Looking into [shuttle-example-go-plan](https://github.com/lunarway/shuttle-example-go-plan/blob/9e5bec02d6bd737e6423d873d95fdab65e7814df/plan.yaml#L23) this should in fact be as you assert.

```suggestion
    fail "Expected output to contain 'something clever', but it was:\n$result"
```